app/app.go

```diff
@@ -152,12 +161,25 @@ func Run(ctx context.Context, conf Config) (err error) {
 		return err
 	}
 
-	log.Info(ctx, "Lock loaded",
+	log.Info(ctx, "Lock file loaded",
+		z.Str("cluster_hash", lockHashHex),
+		z.Str("cluster_name", lock.Name),
 		z.Int("peers", len(lock.Operators)),
 		z.Str("peer_id", p2p.ShortID(tcpNode.ID())),
 		z.Int("peer_index", nodeIdx.PeerIdx),
 		z.Str("enr", localEnode.Node().String()))
 
+	if !conf.TestConfig.DisablePromWrap {
+		// Wrap prometheus metrics with cluster and node identifiers.
+		prometheus.DefaultRegisterer = prometheus.WrapRegistererWith(prometheus.Labels{
```

Comment:
note prometheus [docs](https://prometheus.io/docs/instrumenting/writing_exporters/#target-labels-not-static-scraped-labels) state:
> If you ever find yourself wanting to apply the same label to all of your metrics, stop.